lib/oneview-sdk/resource/api1600/synergy/storage_pool.rb

```diff
@@ -0,0 +1,22 @@
+# (C) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require_relative '../c7000/storage_pool'
+
+module OneviewSDK
+  module API1600
+    module Synergy
+      # Storage pool resource implementation for API 1600 Synergy
```

Comment:
please remove space between API and 1600

Reply:
Done